src/ducks/transactions/TransactionRow.jsx

```diff
@@ -32,8 +32,14 @@ const sAmount = styles['bnk-op-amount']
 const sAction = styles['bnk-op-action']
 
 class _RowDesktop extends React.PureComponent {
-  onSelectTransaction = () =>
+  constructor(props) {
+    super(props)
+    this.onSelectTransaction = this.onSelectTransaction.bind(this)
```

Comment:
Why do you need to change this? Is there a problem with class properties and fat arrows?

Reply:
Yes, it is not possible to mock fat arrow methods in tests.

Reply:
https://github.com/airbnb/enzyme/issues/365#issuecomment-217368222